.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
.idea is something specific to your IDE. Avoid creating a new .gitignore, add this folder and workspace.xml to .gitignore

Reply:
done